.devcontainer/moonbit/devcontainer-feature.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "moonbit",
+  "version": "0.0.1",
+  "name": "moonbitg",
+  "installsAfter": [
+    "ghcr.io/devcontainers/features/common-utils"
+  ]
+}
```

Comment:
What is this for?

Reply:
<img width="183" alt="image" src="https://github.com/user-attachments/assets/36ad1168-ef5e-4c41-be64-fb8bc3f3fcf2">